.github/workflows/master-check.yml

```diff
@@ -1,31 +1,28 @@
-name: Master branch CI
+name: 'Build and Test'
 
-on:
-  push:
-    branches: [ development ]
-    paths-ignore:
-      - '.github/**'
-      - '.gitignore'
-      - 'README.md'
-  pull_request:
-    branches: [ development ]
-    paths-ignore:
-      - '.github/**'
-      - '.gitignore'
-      - 'README.md'
+on: [push]
```

Comment:
[push, pull_request]
Neboj sa procesorového času :).

Reply:
Nad tim jsem taky uvazoval, a prijde mi, ze nemuzes zalozit PR bez toho, aniz bys udelal `push` coz by triggernulo build

Reply:
V pohode - vo https://github.com/cesko-digital/loono/actions/runs/1843385310/workflow to robíme tak, ako som napísal a plní to moje sny :). Prosím uprav a uvidíš.

Reply:
To jsem uz testoval, to se opravdu spusti 2krat, ale budiz, je to tu [e7754b6](https://github.com/cesko-digital/loono-be/pull/46/commits/e7754b658de86c7253844d769198229049e24a1c)